libs/state/src/lib/download/download.action.ts

```diff
@@ -0,0 +1,44 @@
+import { ActionGroup } from '@hra-ui/cdk/state';
+import { DownloadEntry, DownloadFormat, DownloadFormatId } from './download.model';
+
+const Action = ActionGroup('Download');
+
+/**
+ * Register format into state
+ */
+export class RegisterFormat extends Action('Register Format') {
+  constructor(readonly format: DownloadFormat) {
+    super();
+  }
+}
+
+/**
+ * Add entry into download state
+ */
+export class AddEntry extends Action('Add') {
+  constructor(readonly id: DownloadFormatId, readonly entry: DownloadEntry) {
+    super();
+  }
+}
+
+/**
+ * Clear entry from download state
+ */
+export class ClearEntry extends Action('Clear') {
```

Comment:
I rather have an action that clears all entries (i.e. sets entries to the empty object {}) than having to clear each one by one